components/login-portal-layouts/layouts/default/body.html

```diff
@@ -1,21 +1,3 @@
-<!--
```

Comment:
Any specific reason to remove the license header?

Reply:
basically we needed to remove comments from our auth portal shipped content in JSP files. For now we have refactored to hidden comments, but html don't have any kind of feature to hide comments. That's why we decided to remove the license header too in this HTML. Until we find a better way to strip them in the runtime